copier/tools.py

```diff
@@ -50,13 +63,19 @@ def printf_block(e, action, msg="", style=STYLE_WARNING, indent=0, quiet=False):
 no_value = object()
 
 
-def required(value):
+def required(value: Any, **kwargs: Any) -> Any:
```

Comment:
Return type should be identical to the type of parameter `value`.
I have not been able to make it work via `TypeVar`.

Reply:
Got it to work via `TypeVar` now.